src/components/SingleTask.tsx

```diff
@@ -9,24 +9,38 @@ type Props = {
     onAddNote: OnAddNote;
     task: Task;
     handleOnChange: HandleOnChange;
+    OnEditTitle: OnEditTitle;
 };
 
 export default function SingleTask({
     onAddNote,
     task,
     onDelete,
     handleOnChange,
+    OnEditTitle,
 }: Props) {
     const [showInput, setShowInput] = useState(false);
+    const [taskTitle, setTaskTitle] = useState('');
```

Comment:
Why init this with empty string and not with task.title ?
Then just use taskTitle where you now use task.title